opentimelineview/timeline_widget.py

```diff
@@ -255,8 +255,8 @@ def __init__(self, track, *args, **kwargs):
         self._populate()
 
     def _populate(self):
-        for item in self.track:
-            timeline_range = item.trimmed_range_in_parent()
+        for n, item in enumerate(self.track):
+            timeline_range = self.track.trimmed_range_of_child_at_index(n)
```

Comment:
@jminor this is another case where we might want to consider folding transitions back into the regular `item` pattern.

Reply:
I agree.

Reply:
That shouldn't block this change, we can do that later.

Reply:
Agreed.